jest.config.js

```diff
@@ -1,5 +1,10 @@
-module.exports = Object.assign({}, require('./jest.default.config'), {
-  "testPathIgnorePatterns": [
-    "/src/graphql/__tests__"
-  ]
-});
+module.exports = Object.assign(
+  {},
+  require('./jest/jest.default.config.js'),
+  process.env.CI && require('./jest/jest.ci.default.config.js'),
```

Comment:
Despite the fact that I am not agree about confusing syntax for short circuit evaluation, and didn't find a reasonable opinion blaming that approach to write more laconic code, I replaced it to ternary operator. The third operand now an empty object (to be less confusing), but it could be also null (as null and undefined are just ignored in `Object.assign` source arguments).